R/fill_ch_names.R

```diff
@@ -43,187 +58,712 @@ fill_ch_names <- function(ch_data,
       ch_postcode = "AccomPostCodeNo",
       ch_name_validated = "ServiceName",
       ch_date_registered = "DateReg",
-      ch_date_cancelled = "DateCanx"
-    ) %>%
-    dplyr::filter(
-      is.na(.data[["ch_date_cancelled"]]) |
-        (.data[["ch_date_cancelled"]] >= start_fy("1718"))
+      ch_date_cancelled = "DateCanx",
+      ch_active = tidyselect::contains("ServiceStatusAt")
     ) %>%
+    dplyr::filter(is.na(.data[["ch_date_cancelled"]]) |
```

Comment:
Noticed that some places have cancelled dates in 1900. Would it be possible to use date registered or something here? 
Or at least do a wee investigation to make sure that the cases where date cancelled is in 1900 don't match onto anything in Social Care data?

Reply:
Great spot! Will have a look now

Reply:
Added a quick fix on this.